.gitignore

```
@@ -1,4 +1,5 @@
 /local/
+/files/
```

```suggestion
/var/
```

Assuming, that MEDIA_ROOT is set to `BASE_DIR / 'var/media'`.